live-examples/js-examples/weakset-prototype-add.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">var weakset1 = new WeakSet();
+var object1 = new Object();
+
+weakset1.add(object1);
+console.log(weakset1.has(object1));
+// would normally return true
+
+console.log(weakset1.add(1));
+// expected output: "Error: Invalid value used in weak set" in Chrome
+// expected output: "Error: WeakSet value must be an object, got the number 1" in Firefox
```

Comment:
This is tricky, isn't it, because we don't get the first console.log(), which is what soeone might expect. Do you think it's worth doing something like:

```
var weakset1 = new WeakSet();
var object1 = new Object();

weakset1.add(object1);
console.log(weakset1.has(object1));
// expected output: true

try {
  weakset1.add(1);
} catch(error) {
  console.log(error);
  // Error message
}
```
? I'm not sure tbh.

Reply:
I think it makes sense to try and catch errors. Will amend.